apps/passport-server/src/services/issuanceService.ts

```diff
@@ -64,12 +87,16 @@ import { PersistentCacheService } from "./persistentCacheService";
 import { RollbarService } from "./rollbarService";
 import { traced } from "./telemetryService";
 
+export const PUBLIC_KEY_NAME = "Zupass";
```

Comment:
the name of this variable should be something like `ZUPASS_TICKET_PUBLIC_KEY_NAME`